src/zmq/zmqpublishnotifier.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ZMQPUBLISHNOTIFIER_H
```

Comment:
Needs to be ``BITCOIN_ZMQ_ZMQPUBLISHNOTIFIER_H`` also at the header end comment.